acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -6,6 +6,9 @@
 
 # This file contains general test helper methods for pxp-agent acceptance tests
 
+# The standard path for git checkouts is where pcp-broker will be
+GIT_CLONE_FOLDER = '/opt/puppet-git-repos'
```

Comment:
On checking out the branch and poking around I now follow - this is the standard place, as it's now common to the call to `clone_git_repo_on` in acceptance/setup/common/010_Setup_Broker.rb and the `run_pcp_broker` helper here.  Will do for now.